docs/interactive-scroll-view.md

```diff
@@ -0,0 +1,40 @@
+---
+id: interactive-scroll-view
+title: ScrollView 
+sidebar_label: ScrollView
+---
+
+Render `children` elements in a scrollable container with fixed height, using mouse scroll events. Useful for reporters and logs display.
+
+## Props
+
+* `children: React.Element<*>` - Content to be rendered and scrolled.
+* `height: number` - Fixed height of the visible area.
+* `inputStream: tty$ReadStream` (default: `process.stdin`) - Input stream to capture scroll events from.
+* `outputStream: tty$WriteStream` (default: `process.stdout`) - Output stream to enable/disable mouse reporting, hence toggling scroll events.
+* `reportingRatio: number` (default: `0.25`) - Mouse scrolling reporting ratio, effectively controlling the speed of scroll.
```

Comment:
Since this controls scroll speed, maybe call it `speed`? `reportingRatio` seems like an implementation detail and can be confusing to the consumer.

Reply:
I've to admit that I've also found `reportingRatio` quite abstract. Speed would work better for me. 👍

Reply:
But it's not speed as well. Speed is distance per unit of time and here the distance is always equal to 1, so it's like a frequency.

Reply:
Frequency of when the scroll event is fired? Anyway, that's still an implementation detail. You don't have to keep the value same. You can also change the value to mean speed and internally convert it to frequency or whatever.

Reply:
👎 for `speed`. I'm fine with different word but not `speed`

Reply:
I suggested speed because you wrote that it controls speed. Change it to whatever makes sense. The question to ask is, what's the user gonna use it for?

Reply:
Does `scrollSensitivity` make sense? Also the description is quite uninformative tbh, at least for me. What happens if it's 0.2 and what if 0.8? Based on the quick code glance I guess the higher it is, the more scroll events you'll get. An example would be nice to have :)

Reply:
It does